lib/main_common.pm

```diff
@@ -568,15 +568,18 @@ sub load_jeos_tests {
     load_boot_tests();
     loadtest "jeos/firstrun";
     loadtest "jeos/record_machine_id";
+    loadtest "console/system_prepare" if is_sle;
     loadtest "console/force_scheduled_tasks";
     unless (get_var('INSTALL_LTP')) {
-        loadtest "jeos/grub2_gfxmode";
+        loadtest "jeos/grub2_gfxmode" if get_var('CONTAINER_RUNTIME');
         loadtest "jeos/diskusage";
         loadtest "jeos/build_key";
-        loadtest "console/prjconf_excluded_rpms";
+        loadtest "console/prjconf_excluded_rpms" if get_var('CONTAINER_RUNTIME');
+    }
+    unless (get_var('CONTAINER_RUNTIME')) {
+        loadtest "console/journal_check";
+        loadtest "microos/libzypp_config";
     }
-    loadtest "console/journal_check";
```

Comment:
Well, this has been running in O3 so far.. https://openqa.opensuse.org/tests/2095447#step/journal_check/1
But I don't have strong feelings to remove it, we already do that check in other jeos specific tests.

Reply:
i think this is a good catch. i believe both lines are removed by accident. will fix

Reply:
oh they was not removed. just noticed. i got confused seeing the comment in this part of the diff. Soo is it ok based on your last sentence? i am with you on it.

Reply:
`journal_check` makes more sense to be run in the end, after all the tests... so, let's just skip it for container tests for now.. we can always enable it in main_containers... let's see. So, OK with this change.

Reply:
can we resolve this?